docs/book/installation/sylius_plus_installation.rst

```diff
@@ -40,8 +40,10 @@ Installing Sylius Plus as a plugin to a Sylius application
 .. code-block:: bash
 
     composer config repositories.plus composer https://sylius.repo.packagist.com/ShortNameOfYourOrganization/
-    composer require sylius/plus --no-update
+    composer require symfony/dependency-injection
+    composer require friendsofsymfony/oauth-server-bundle:2.0.0-alpha.0
```

Comment:
```suggestion
    composer require "friendsofsymfony/oauth-server-bundle":">2.0.0-alpha.0 ^2.0@dev"
```

Reply:
What is more, this requirement is needed only on Sylius 1.10, because in Sylius 1.9, it should be already installed